media-gfx/blender/blender-2.83.15.ebuild

```diff
@@ -153,8 +153,8 @@ src_unpack() {
 	fi
 
 	if use test; then
-		mkdir -p lib/tests || die
-		mv "${WORKDIR}"/blender*tests* lib/tests || die
+		mkdir -p lib || die
+		mv "${WORKDIR}"/blender*tests/tests lib || die
```

Comment:
I'm not sure what you are suggesting here.
What does this have to do with the SRC_URI?

Reply:
SRC_URI has `${PN}-2.83.1-tests`, which is what the `blender*tests` glob is matching, rather than using that glob a variable could be shared between the two.